docs/Documentation/Scripting/Building-Blocks/Events-List.md

```diff
@@ -738,3 +738,20 @@ events:
   events:
     cancel: "cancelconversation"
 ```
+## Log message to console: `log`
+
+**persistent**, **static**  
+
+Prints a provided message to the server log. Any variables used in the message will be resolved. 
+Note that when used in static context (by schedules) replacing variables won't work as the event is player independent.
```

Comment:
This is only true for variables that need the player as reference, but some also work in static context.